src/Services/HardwareService.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Taecontrol\Larvis\Services;
+
+use Taecontrol\Larvis\Exceptions\CpuHealthException;
+use Taecontrol\Larvis\Exceptions\DiskHealthException;
+use Taecontrol\Larvis\ValueObjects\Data\HardwareData;
+use Taecontrol\Larvis\Exceptions\MemoryHealthException;
+
+class HardwareService
+{
+    public function getHardwareData()
+    {
+        $cpuLoad = $this->getCpuLoadUsage();
+        $memory = $this->getMemoryUsage();
+        $disk = $this->getDiskUsage();
+
+        $HardwareData = new HardwareData($cpuLoad, $memory, $disk);
+
+        return $HardwareData->toArray();
+    }
+
+    public function getDiskUsage()
+    {
+        $result = false;
+        $freeSpace = false;
+        $totalSpace = false;
+
+        if (function_exists('disk_free_space') && function_exists('disk_total_space')) {
+            $freeSpace = round((disk_free_space('/') / pow(1024, 3)), 1);
+            $totalSpace = round((disk_total_space('/') / pow(1024, 3)), 1);
+        }
```

Comment:
Don't create boolean variables and then assign them number values

Reply:
You should wrap this code using a try/catch, in case there's an error it can return the exception. The same with getMemoryUsage and getCpuLoadUsage